_clanky/nové-povinnosti-pro-obce-kraje-a-orgány-státní-správy-v-oblasti-otevřených-dat.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Nové povinnosti pro obce, kraje a orgány státní správy v oblasti otevřených dat
+detail: true
+ref: Nové-povinnosti-pro-obce-kraje-a-orgány-státní-správy-v-oblasti-otevřených-dat
+lang: cs
+image: ../attachments/články/Nové-povinnosti-pro-obce-kraje-a-orgány-státní-správy-v-oblasti-otevřených-dat/úvod.webp
```

Comment:
Názvy adresářů je lepší pojmenovávat malým písmenem, pokud to není zkratka

Reply:
Adresář tam nebude, protože nebude obrázek - je tam v úvodu dost textu, tak aby nám to nerozházelo layout webu.